README.md

```diff
@@ -30,48 +30,92 @@ java -jar app-v-0.1.0.war
 ```
 
 
-<!--
+
  ## Running in Docker
 
+
 To run the application using Docker you have to install Docker and docker-compose on your local computer.  To install Docker see [here](https://docs.docker.com/get-docker/) and to install docker-compose see [here](https://docs.docker.com/compose/install/). 
 
-Assuming you have installed both Docker and docker-compose, to run the Onto Viewer application together with its front-end on your local machine, please make sure that you have the following folder setup:
+Create `onto-viewer-web` and `strapi-dashboard` directories next to onto-viewer root folder.
+Install Strapi with the development database in the `strapi-dashboard` directory and copy the general templates in nuxt from `general` directory into `onto-viewer-web` and configure then according to the instructions [README.md](https://github.com/edmcouncil/html-pages/blob/develop/general/README.md). 
+Replace `nuxt.config.js` from `onto-viewer-web` to `\onto-viewer\docker\web`. 
+
+In the `onto-viewer/docker/runtime/server/` folder, you must put configuration and ontologies files.  You can find samples of these files in the [onto-viewer-config-loader/src/main/resources](https://github.com/edmcouncil/onto-viewer/tree/update-readme-and-docker-files/onto-viewer-config-loader/src/main/resources).  Note that the `onto-viewer/docker/runtime/` folder is excluded from Git, so you can freely put there any file you want.
+
+Assuming you have installed Docker and docker-compose, installed and configured Strapi and general html templates to run the Onto Viewer application together with its front-end on your local machine, please make sure that you have the following folder setup:
```

Comment:
I suggest the following change:
***
1. Create `onto-viewer-web` and `strapi-dashboard` directories next to onto-viewer root folder.
2. Install Strapi with the development database in the `strapi-dashboard` directory. To this end,  run `npx create-strapi-app@latest strapi-dashboard --quickstart` command.
3. Copy files and folders from [html-pages repository](https://github.com/edmcouncil/html-pages):
- `html-pages/general/strapi/.tmp/ > strapi-dasboard/.tmp/`
- `html-pages/general/strapi/src/ > strapi-dasboard/src/`
- `html-pages/general > onto-viewer-web/`
- `html-pages/general > strapi-dasboard/Dockerfile`
4. Replace:
`onto-viewer-web/nuxt.config.js > \onto-viewer\docker\web\nuxt.config.js`
5. In the `onto-viewer/docker/runtime/server/` folder, you must put configuration and ontologies files. You can find samples of these files in the [onto-viewer-config-loader/src/main/resources](https://github.com/edmcouncil/onto-viewer/tree/update-readme-and-docker-files/onto-viewer-config-loader/src/main/resources). Note that the `onto-viewer/docker/runtime/` folder is excluded from Git, so you can freely put there any file you want.

The final folder system should look like shown below: